components/ui/cardHorizontal/index.tsx

```
@@ -22,19 +23,30 @@ const AdCard = dynamic(() => import("@components/ui/adCard"), {
   ssr: false,
 });
 
-function CardHorizontal({ event, isLoading, isPriority }) {
+const CardHorizontal: React.FC<CardHorizontalProps> = ({
```

ESLint

[error] 'React' is not defined.

(no-undef)